src/screen/status.rs

```diff
@@ -249,6 +254,36 @@ fn create_status_section_items<'a>(
     .chain(items::create_diff_items(config, diff, &1, true))
 }
 
+fn create_stash_list_section_items<'a>(
+    config: Rc<Config>,
+    repo: &Repository,
+    header: &str,
+) -> impl Iterator<Item = Item> + 'a {
+    let stashes = items::stash_list(&config, repo, 10).unwrap();
+    if stashes.is_empty() {
+        vec![]
+    } else {
+        let style = &config.style;
+        vec![
+            Item {
+                display: Line::raw(""),
+                depth: 0,
+                unselectable: true,
+                ..Default::default()
+            },
```

Comment:
This function could save some space. Saw that there's more of these in this module too.
```suggestion
            items::blank_line(),
```